### DISCUSSION TITLE
Truncation of Resolved Comments during Pull Request Code Review

### DISCUSSION THREAD

**Discussion by NTKonzen** [community/community#16111] [Pull Requests]
Currently having an issue with a large PR with many comments. As comments are addressed, they are marked "resolved" which truncate the comment. Awesome. Been running into an issue though with the full comment list truncation as seen below:
![Screen Shot 2022-05-05 at 2 04 23 PM](https://user-images.githubusercontent.com/66763982/167025327-68a7c747-7a91-4e57-8d04-f83ede3e2cb8.png)
The problem is, that this truncated section of the list included several unresolved comments and there is no indication of such. Is there a way that resolved comments can get priority for list truncation so that unresolved comments are always seen? Or is there a way we can indicate in that little "hidden conversations" box that there are unresolved comments within?

Thank you!

**Comment by bugfood** [comment]
This is pretty annoying and I would rather not have this comment-collapsing feature at all.

Last week, a code author missed some of my review comments because they were hidden, and I had to point this out in another round of review.

Today I wrote a review, and three of my comments are now already hidden by default.
![image](https://user-images.githubusercontent.com/2976480/224868720-4dd7af9e-ea7e-42bd-9e37-5a01a6fda9e0.png)
Github has no way of knowing whether these comments are unimportant.

Edit: sorry this shows up as a "suggested answer". I don't have an answer to suggest. All I could find was partial workarounds:
https://stackoverflow.com/questions/63883807/disable-hidden-conversations-in-github-pull-requests
https://webapps.stackexchange.com/questions/148308/how-to-always-show-all-pr-conversations-in-github

**Comment by ChrisMacGregorONI** [comment]
https://github.com/orgs/community/discussions/130618#discussioncomment-12105463
